### ISSUE TITLE
File moves very slowly.......

### ISSUE THREAD

**Issue by SiamSami** [linuxdeepin/dde-file-manager#84]
Hello! I have a computer of 4 x 3.30 ghz cpu and 8 gb ram. Yet when I try to copy a file from one place to another the speed is dead slow.....!

The speed is about 20 mb/s or sometimes 60 kb/s or even less !
Please fix this issue as soon as possible.........

**Comment by aiamuzz** [comment]
Yes even i face this issue ... i work with images with long file names(by the 100's and 1000) ...  ... a copy or move operation is so slow ... even though the size is just a few MB's ...

The exact same operation when performed on my Windows 7 VM ... it happens in a flash.

Would really love the speed of copy and move in Windows to rub off to our Deepin File Manager too.
